macos/FluentUI/Button/Button.swift

```diff
@@ -247,6 +279,30 @@ open class Button: NSButton {
 			layer.backgroundColor = backgroundColorRest?.cgColor
 			layer.borderColor = borderColorRest?.cgColor
 		}
+
+		if usesBorderShadows {
```

Comment:
Per below comment, rather than checking this, we can check if the specific layers we want to update are non null using if/lets. Just a tiny bit more direct - if the layers we want to modify exist, then we are free to operate on them